packages/phrases/src/locales/zh-cn.ts

```diff
@@ -406,7 +406,15 @@ const translation = {
       },
       setup_warning: {
         setup: '设置',
-        no_connector: '你尚未添加社交登录连接器。你需要先完成设置才能启用。',
+        no_connector: '',
```

Comment:
why remove the content of `no_connector`?

Reply:
Splitted into sub items.